mock-core-configs/etc/mock/templates/centos-stream-10.tpl

```diff
@@ -34,7 +34,99 @@ user_agent={{ user_agent }}
 name=CentOS Stream $releasever - Koji Local - BUILDROOT ONLY!
 baseurl=https://kojihub.stream.centos.org/kojifiles/repos/c{{ releasever }}s-build/latest/$basearch/
 cost=2000
-enabled=1
+enabled=0
 skip_if_unavailable=False
 
+[baseos]
+name=CentOS Stream $releasever - BaseOS
+baseurl=https://composes.stream.centos.org/stream-10/production/latest-CentOS-Stream/compose/BaseOS/$basearch/os
+#metalink=https://mirrors.centos.org/metalink?repo=centos-baseos-$stream&arch=$basearch&protocol=https,http
+gpgkey=file:///usr/share/distribution-gpg-keys/centos/RPM-GPG-KEY-CentOS-Official-SHA256
```

Comment:
This should use the versioned key path...
```suggestion
gpgkey=file:///usr/share/distribution-gpg-keys/centos/RPM-GPG-KEY-CentOS-$releasever
```

Reply:
c9s config doesn't

Reply:
But happy to do that if this is what the mock maintainers want.

Reply:
While it would be cleaner, I don't think it is completely necessary. :shrug: